src/TConsole.h

```diff
@@ -161,6 +161,8 @@ class TConsole : public QWidget
     void showEvent(QShowEvent* event) override;
     void hideEvent(QHideEvent* event) override;
     void setConsoleBgColor(int, int, int, int);
+    QColor getConsoleBgColor() const { return mBgColor; };
```

Comment:
This *getter* is redundant - the member it is returning is **`public:`** and the only reason for the setter above it is that that takes the individual components, generates a `QColor` from them and passes them to the pair of associated `TTextEdit`` instances before calling other methods...

Reply:
Technically it is, but it'll be easier for others to read the code if it is consistent.